src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,24 @@
 package mate.jdbc;
 
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer manufacturer = new Manufacturer("Mazda", "Japan");
+        manufacturer = manufacturerDao.create(manufacturer);
+        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(3L);
```

Comment:
```suggestion
        manufacturer = manufacturerDao.create(manufacturer);
        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(manufacturer.getId());
```
If we want to check if manufacturer is created, we can use `manufacturer.getId()` then we don't need to change `3L` manually every time